src/index.js

```diff
@@ -3,6 +3,19 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import * as firebase from 'firebase/app';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyByClHaWsJw2jVjIlTQ2FIzaeI6hs0Y7tk',
```

Comment:
Is it something we have to treat as a secret? In some tutorials I saw people hiding those keys.

Reply:
I would say so.

Reply:
no, these aren't secret configs. I can add them as environment variables instead of keeping them like this, but in general, as long as we add read/write rules to the database (depending on whether or not users are authenticated) this should be fine.

Reply:
I found this: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public

Reply:
So if we secure the app from console (e.g. disallow to read DB only with api key) then revealing api key won't expose personal data

Reply:
@semiluna could you please take care of those read/write rules to the db?

Reply:
Yes, I can add rules right now, but the problem with that is that before we add a "sign in" workflow, nobody will be able to play around with the database - to test the API calls with fake data, for example.

Reply:
I'd just leave this DB open for development purposes and as soon as sign in workflow is added (which is btw needing a read access to the db to check membership status) we can discuss how to secure data. Is it ok?